Achievements/SelfMade.lua

```diff
@@ -1,53 +1,92 @@
 local _G = _G
 local self_made_achievement = CreateFrame("Frame")
 _G.achievements.SelfMade = self_made_achievement
+local player = UnitName("Player")
 
 -- General info
 self_made_achievement.name = "SelfMade"
 self_made_achievement.title = "Self-Made"
 self_made_achievement.class = "All"
 self_made_achievement.icon_path = "Interface\\Addons\\Hardcore\\Media\\icon_self_made.blp"
 self_made_achievement.description =
-	"Complete the Hardcore challenge without at any point equipping an item that you have not crafted yourself. Items your character has conjured (e.g. Firestones) are considered crafted. No items bought, dropped, or rewarded by quests are allowed to be equipped (items provided for a quest can be equipped). The items your character starts with are allowed to be equipped. Bags are equipped items."
+"Complete the Hardcore challenge without at any point equipping an item that you have not crafted yourself. Items your character has conjured (e.g. Firestones) are considered crafted. No items bought, dropped, or rewarded by quests are allowed to be equipped, fishing poles MAY be equipped, starting items MAY be re-equipped if taken off (items provided for a quest can be equipped). The items your character starts with are allowed to be equipped. Bags are equipped items."
+local Combine = {
```

Comment:
One cool trick when doing comparisons is to put the items as keys in a table instead of a list 

```
local Combine = { [35] = 1, [36] = 1, ...} 
```

The 1's don't really matter but instead of having to loop through all the items with ipairs you can just do

```
if Combine[item_id] ~= nil then -- Exists it in list
else -- Doesn't exist in list
end
```

Reply:
Neat trick! Will need to remember that for later 👯